javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,49 @@
+let boughtCandyPrices = [];
+let price = 0;
+let i = 0;
+let totPrice = 0;
+let amountToSpend = Math.floor(Math.random() * 100);
+addCandy = (candyType, weight) => {
+  switch (candyType) {
+    case "Sweet":
+      price = weight * 0.5;
+      break;
+    case "Chocolate":
+      price = weight * 0.7;
+      break;
+    case "Toffee":
+      price = weight * 1.1;
+      break;
+    case "Chewing-gum":
+      price = weight * 0.03;
+      break;
+
+    default:
+      "You are not entering a candy!";
+  }
+
+  boughtCandyPrices.push(candyType, price);
+};
+
+canBuyMoreCandy = () => {
+  while (i < boughtCandyPrices.length) {
+    totPrice += boughtCandyPrices[1];
+    i++;
+  }
+  console.log(totPrice + " VS " + amountToSpend);
+
+  if (totPrice <= amountToSpend) {
+    console.log("You can buy more, so please do!");
+  } else {
+    console.log("Enough candy for you!");
+  }
+};
```

Comment:
I am using while loop, because it s a hint from homework description! Calculating total  price.                                                                    

_**Hint: Use a for loop to calculate the total price of the candy pieces.
Optional Use a while loop instead of a for loop.**_